tokio/Cargo.toml

```diff
@@ -106,6 +106,7 @@ socket2 = { version = "0.6.0", optional = true, features = ["all"] }
 # Requires `--cfg tokio_unstable` to enable.
 [target.'cfg(tokio_unstable)'.dependencies]
 tracing = { version = "0.1.29", default-features = false, features = ["std"], optional = true } # Not in full
+usdt = { version = "0.6.0", default-features = false, optional = true } # Not in full
```

Comment:
This shouldn't be a feature flag (because once they're in it's a breaking change for them to get removed, even if `tokio_unstable` isn't enabled.

Instead we should require 2 config flags: `tokio_unstable` and `tokio_usdt`.

This is similar to `tokio_taskdump`.

Reply:
Fair 👍

Reply:
You're right. Things have changed and my memory appears to be wrong as to the initial reason. So this can stay as it was. Sorry.